src/commands/audioflags.ts

```diff
@@ -551,7 +551,17 @@ class Flag {
  * A enum representing a Tone/Note to be used for Audio Flags.
  */
 enum Tone {
-    D2 = "D2",
-    D4 = "D4",
-    D6 = "D6"
+    Piano1 = "D2",
+    Piano2 = "D4",
+    Piano3 = "D6",
+    Violin1 = "E2",
+    Violin2 = "E4",
+    Violin3 = "E6",
+    Guitar1 = "F2",
+    Guitar2 = "F4",
+    Guitar3 = "F6",
+    Marimba1 = "G2",
+    Marimba2 = "G4",
+    Marimba3  = "G6"
+
 }
```

Comment:
Replace this enum with a FlagSound class with the following fields:
- name
- note
- instrument
all with the type of String.

This will require changes in the following functions/data structures:
- showAudioFlagQuickPick (the way the name is obtained from the class will need to be adjusted)
- getAudioFlagToneFromLineNumber (this function should return the FlagSound class rather than a string)
- Flag class (the note field should be changed to something like "sound" with the type of FlagSound)